apps/full-site-editing/full-site-editing-plugin/donations/tabs.js

```diff
@@ -0,0 +1,88 @@
+/**
+ * External dependencies
+ */
+import classNames from 'classnames';
+
+/**
+ * WordPress dependencies
+ */
+import { __experimentalBlock as Block } from '@wordpress/block-editor';
+import { Button } from '@wordpress/components';
+import { useEffect, useState } from '@wordpress/element';
+import { __ } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import Context from './context';
+import Controls from './controls';
+import Tab from './tab';
+import StripeNudge from './stripe-nudge';
+
+const Tabs = ( props ) => {
+	const { attributes, products, setAttributes, stripeConnectUrl } = props;
+	const { oneTimePlanId, monthlyPlanId, annuallyPlanId } = attributes;
+	const [ activeTab, setActiveTab ] = useState( 'one-time' );
+
+	const isTabActive = ( tab ) => activeTab === tab;
+
+	const tabs = {
+		'one-time': { title: __( 'One-Time', 'full-site-editing' ) },
+		...( monthlyPlanId && { '1 month': { title: __( 'Monthly', 'full-site-editing' ) } } ),
+		...( annuallyPlanId && { '1 year': { title: __( 'Yearly', 'full-site-editing' ) } } ),
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **58** times:
`translate( 'Yearly' )` *ES Score: 15*
See 1 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/donation/customize-messages)